easistrain/EDD/fitEDD.py

```diff
@@ -59,7 +66,7 @@ def fitEDD(
                     raise KeyError(
                         f"{positioner} was not found in input positioners! Possible values: {input_positioners.keys()}"
                     )
-                pos_data = input_positioners[positioner][()]
+                pos_data = input_positioners[positioner][detectorSliceIndex]
```

Comment:
Almost. The positioners either have as many points as the detectors (i.e. they are motors used in the scan) or they are scalars. You'll need to handle both cases.